src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.search {
+  display: flex;
+  box-sizing: border-box;
+  height: 70px;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
```

Comment:
This is a bit of wrong box-shadow, I think that is why your tests fall.
Use this - `0 1px 8px 0 rgba(61, 78, 97, 0.10);`